src/hooks/useReportIDs.tsx

```diff
@@ -118,7 +79,7 @@ function ReportIDsContextProvider({
     currentReportIDForTests,
 }: ReportIDsContextProviderProps) {
     const [priorityMode] = useOnyx(ONYXKEYS.NVP_PRIORITY_MODE, {initialValue: CONST.PRIORITY_MODE.DEFAULT});
-    const [chatReports] = useOnyx(ONYXKEYS.COLLECTION.REPORT, {selector: chatReportSelector});
+    const [chatReports] = useOnyx(ONYXKEYS.COLLECTION.REPORT);
```

Comment:
I think we will still have that field, see the code below which is removed:

```ts
        errorFields: {
            addWorkspaceRoom: report.errorFields?.addWorkspaceRoom,
        },
```

We are accessing `addWorkSpaceRoom` from `errorFields?`, so even if we remove the selector this will still be accessible. However, to gain more confidence can you point me how we can create some error fields?

I removed the selector because it was adding relatively less value and since we pass a selector `useOnyx` will be using `deepEqual` under the hood, which accounts for the execution time.

Reply:
@mountiny I also have the same question, how can I make an API failed? I tried to turn on `Simulate failing network requests` option, but I don't think it works.

Reply:
Yeah I think you can try to get the request to fail

Or force in a space into the room name by hardcoding it, that should make the request to fail

Reply:
So I tried toying around with `createDraftInitialWorkspace` which is the function called when clicking on New Workspace button. I tried adding space, special characters, commenting out some fields but I didn't get any `errorFields` 🤔  I also tested this with selector in place but the result is the same. Which means that nothing is failing and we can't see `errorFields`.

I have also tried failing the API request for the following but no luck:

```ts
function createWorkspace(policyOwnerEmail = '', makeMeAdmin = false, policyName = '', policyID = generatePolicyID()): CreateWorkspaceParams {
    const {optimisticData, failureData, successData, params} = buildPolicyData(policyOwnerEmail, makeMeAdmin, policyName, policyID);
    API.write(WRITE_COMMANDS.CREATE_WORKSPACE, params, {optimisticData, successData, failureData});

    return params;
}
```

Then I added the `errorFields` in fake report which we generate for unit tests and I get the `errorFields` correctly from `useOnyx`. So probably we are not sure how to populate `errorFields` from the App and we can only simulate this as of now.

<img width="949" alt="Screenshot 2024-06-25 at 6 57 21 PM" src="https://github.com/Expensify/App/assets/47336142/196af781-aa67-4a46-8dbc-b33a8a84731b">

Reply:
you can try to delete few parameter here you will see the error:
https://github.com/Expensify/App/blob/2f688ad31969f148b18a3793b057b31c4f02aec7/src/libs/actions/Policy/Policy.ts#L1684-L1696

Reply:
@hungvu193  Already tried this but all the same.

Reply:
@mountiny  Thank you 😄 After trying these steps, I am able to get `errorFields.addWorkspaceRoom` in `chatReports` without selector.

![Screenshot 2024-06-27 at 3 09 12 PM](https://github.com/Expensify/App/assets/47336142/c99159c4-8183-4c88-b874-ae0f0587b2f4)